src/lib/viewers/controls/page/PageControls.tsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import Browser from '../../../Browser';
+import { BROWSERS } from '../../../constants';
+import { decodeKeydown } from '../../../util';
+import fullscreen from '../../../Fullscreen';
+import IconArrowDown24 from '../icons/IconArrowDown24';
+import IconArrowUp24 from '../icons/IconArrowUp24';
+import './PageControls.scss';
+
+export type Props = {
+    onPageChange: (arg0: number) => void;
+    pageCount: number;
+    pageNumber: number;
+    viewer: HTMLElement;
+};
+
+export default function PageControls({ onPageChange, pageCount, pageNumber, viewer }: Props): JSX.Element {
+    const isSafariFullscreen = Browser.getName() === 'Safari' && fullscreen.isFullscreen();
+    const [isInputShown, setIsInputShown] = React.useState(false);
+    const [tempPageNumber, setTempPageNumber] = React.useState(pageNumber.toString());
+    const inputEl = React.useRef<HTMLInputElement>(null);
+
+    React.useEffect(() => {
+        if (inputEl && inputEl.current) {
+            setTempPageNumber(pageNumber.toString());
+            inputEl.current.focus();
+        }
+    }, [isInputShown, pageNumber]);
+
+    const onPageNumInputChange = (e: React.ChangeEvent<HTMLInputElement>): void => {
+        setTempPageNumber(e.target.value);
+    };
+
+    const onPageNumInputBlur = (): void => {
```

Comment:
We use the `on/handle` naming convention for event props, so something like the following would work:

```
<input
    onBlur={handleInputBlur}
    onChange={handleInputChange}
/>

Reply:
Got it, will make the change!